src/test/java/gatlingdemostore/pageobjects/Checkout.java

```diff
@@ -0,0 +1,25 @@
+package gatlingdemostore.pageobjects;
+
+import static io.gatling.javaapi.core.CoreDsl.*;
+import static io.gatling.javaapi.http.HttpDsl.*;
+
+import io.gatling.javaapi.core.*;
+
+public class Checkout {
+
+  public static ChainBuilder viewCart =
+      doIf(session -> !session.getBoolean("customerLoggedIn"))
+          .then(exec(Customer.login))
+          .exec(
+              http("Load Cart Page")
+                  .get("/cart/view")
+                  .check(status().is(200))
+                  .check(css("#grandTotal").is("$$${cartTotal}")));
```

Comment:
Oh right, the $ escaping :( Have you verified this still works?

Reply:
Isn't that dollar escaping only needed when using string interpolation in scala? Which is not used in the scala version as well... So I've assume that we were trying for a cartTotal of `10` to create string `$$10` instead of `$10`...

Reply:
Do a `$#{cartTotal}` instead